persistence/sql/batch/create.go

```diff
@@ -0,0 +1,139 @@
+// Copyright © 2023 Ory Corp
```

Comment:
This is some complex code that has no appropriate proof (except for circumstantial because some other method invokes it somewhere) that it works. It also does not have tests ensuring we won't introduce regressions. Please add tests.

Reply:
This is now tested through handler and persister tests as well as tests for the individual methods.